config.php

```diff
@@ -33,7 +35,147 @@
 	// - repo: the repo to move the issue to
 
 	'actions' => [
-		[
+
+        /**
+         * Status: Need more info
+         */
+        [
+			'label' => 'status:need more info',
+			'comment' => <<<COMMENT
+Thanks for posting your issue in our issue tracker. 
+In order to properly assist you, we need more information. 
+
+Can you supply us with answers to the following questions:
+* When does the issue occur?
+* What do you see?
+* What was the expected result?
+* (optional) Can you supply us with a stacktrace?
+* (optional) How could a unit test look like proving your issue?
+
+Thanks!
+
+_This is an automated comment, triggered by adding the label `status:need more info`._
+COMMENT
+			,
+        ],
+
+        [
+			'label' => 'status:need more info (expired)',
```

Comment:
correct. After labeling manually. Dont see how the bot could do this automatically.